frontend/src/views/AccountView.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <div class="w-full mt-auto mb-auto ml-3 mr-3">
+    <Suspense>
+      <AccountDetailsEditContainer :showadminview="true" />
```

Comment:
We probably shouldnt have the admin view in the regular account editor

Reply:
yeah that would be bad

Reply:
since i want to test the component also for Adminview this will stay until just before merge. But you could review again, since i did some major changes @c0derMo